### DISCUSSION TITLE
Strong positive feedback for private profiles

### DISCUSSION THREAD

**Discussion by milesmcc** [community/community#21107] [Other Feature Feedback, Questions, & Ideas]
Just wanted to give a huge shoutout to the folks at GitHub who advocated for private profiles, and everyone who helped push the change through.

There are a lot of reasons why folks might want to set their profiles to private — from safety and legal issues to personal preference — and I'm really grateful that the option exists. Kudos!

**Comment by thuongtruong1009** [comment]
Hi @milesmcc 
I'm like you, I'm very grateful to the people who have supported me, and to the open source community.
I am also looking for a partner for cooperation in the future. Hope to connect with more people all over the world.
Cooperate with me at [my profile](https://github.com/thuongtruong1009)
Thank you very much!
